client-core/src/service/key_service.rs

```diff
@@ -21,32 +21,32 @@ where
         KeyService { storage }
     }
 
-    /// Adds a new public-private keypair to storage
-    pub fn add_keypair(
+    /// Adds a new wallet_name-private keypair to storage
```

Comment:
I think it needs to store multiple key pairs for one wallet.

Reply:
mulitiple key pairs (for transfer addresses and staking address ) are stored in Wallet.

Reply:
So I think `KeyService` can be removed now since key pairs are in `Wallet`?

Reply:
but the private key is not stored in Wallet, how do you think we move all the associated items (like key pairs, private key, multisign_address) into the `Wallet`, because all the `items`  can be accessed with `enckey`(+ wallet_name) just like `Wallet`, we currently stored them in different place, if we want to get them, we first get the `Wallet`, then use the key in the wallet to get the value, it's not convenient.